content/en/docs/gke/pipelines/upgrade.md

```diff
@@ -42,6 +42,10 @@ However, there's no current support to upgrade from Kubeflow 1.0 or earlier to K
 
 Upgrade Support for AI Platform Pipelines is in **Alpha**.
 
+{{% alert title="Warning" color="warning" %}}
+Although Kubeflow Pipelines Standalone deployments also show up in AI Platform Pipelines dashboard, DO NOT follow instructions below. Because data is deleted by default when a Kubeflow Pipelines Standalone deployment is deleted.
```

Comment:
Suggested revision:

Kubeflow Pipelines Standalone deployments also show up in the AI Platform Pipelines dashboard, DO NOT follow instructions below if you deployed Kubeflow Pipelines using standalone deployment.